client/components/mma/accountoverview/updateAmount/SupporterPlusUpdateAmountForm.tsx

```diff
@@ -304,6 +312,7 @@ export const SupporterPlusUpdateAmountForm = (
 							>
 								<TextInput
 									label={otherAmountLabel}
+									data-cy="supporter-plus-other-amount-input"
```

Comment:
We can't target this using the label? (Feels a bit closer to how a human would interact with this).

Reply:
yeah good shout, I aggree ... the new selector in the test for the input is a bit more involved but I guess the benefit is that we're not polluting the dom and we're closer to how a user would navigate the site